_config.yml

```diff
@@ -27,7 +27,7 @@ asciidoctor:
   - UserCLIBookName=CLI Guide
   - RulesDevBookName=Rules Development Guide
   - EclipseCrsGuideTitle=Eclipse and Red Hat CodeReady Studio Plugin Guide
-  - IntroToMTABookName=Introduction to the Migration Toolkit for Applications
+  - IntroTo{ProductShortName}BookName=Introduction to the Migration Toolkit for Applications
```

Comment:
Does this work as expected? If it does, great. Otherwise, change it back to MTA.

Reply:
@RichardHoch 
How do I check that?